### ISSUE TITLE
Auto-checkin should not display shout dialog.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [iamakshayv/foursquared#22]
```
Basically I would describe this as "overlapping modal dialogs on badge
achievement result in disappearing checkin dialog".
```

Original issue reported on code.google.com by `jlapenna` on 8 Aug 2009 at 10:17

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 5ba6cffd01.
```

Original comment by `jlapenna` on 9 Aug 2009 at 4:14
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 5ba6cffd01.
```

Original comment by `jlapenna` on 5 Jan 2010 at 5:46
